### PR TITLE
[DEVHUB-1314] Add Prometheus monitoring to devcenter Next.js app

### PR DESCRIPTION
**Pull request by ajhenri**
Reuse metrics endpoint that was created in another ticket by Jacky. Defines configuration rules for metrics monitoring and sets up basic Alertmanager rules for service uptime and container readiness.

Update staging CDN link based on Kanopy advisory.

Example query:
`up{namespace="devrel", job="devcenter-frontend-ah-web-app"}`

### CODE REVIEW

File: src/pages/api/metrics.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { Registry } from 'prom-client';
+import { prometheus } from '../../config/prometheus';
+
+const metricsApi = async (req: NextApiRequest, res: NextApiResponse) => {
+    const authHeader = req.headers.authorization;
+    if (!authHeader) return res.status(400).send('Bad Request');
```
Comment (mcolella14) [inline src/pages/api/metrics.ts:7]: This is incredibly nitpicky, but I think it should technically be a 401 too since it's an unauthorized request.

**Review by mcolella14 — APPROVED** [review]
Looks great! Will feel better knowing we have some alerts now.

**Review by gfunk1230 — APPROVED** [review]

File: src/pages/api/metrics.ts (hunk 1 above)
Comment (ajhenri) [reply src/pages/api/metrics.ts:7]: Not nitpicky at all, thanks for spotting!

**Review by jackyw2018 — APPROVED** [review]
LGTM

**Pull request merged**